### PR TITLE
Patch relnotes.k8s.io to release v1.18.1

### PR DESCRIPTION
**Pull request by puerco**
Automated patch to update relnotes.k8s.io to k/k version `v1.18.1`

### CODE REVIEW

**Comment by puerco** [comment]
This is the first automated PR to update the release notes website created directly from krel . 

I'm generating the notes for `v1.18.1` to test. I will send a PR with the changes to krel which I hope we can merge shortly to generate the `v1.19-beta.0` release notes.

I'd like to hear if you have any comments on the PR body. It is rather short because I'm not sure what to put in there.

This PR was created with the following krel invocation:
```
krel release-notes --create-website-pr --tag v1.18.1 \
   --website-org=puerco --website-repo=release-notes
```
Any comments or suggestions on how to improve the PR are welcome. 
/cc @justaugustus @saschagrunert 
/cc @reylejano-rxm @JamesLaverack @AGMETEOR @wilsonehusin

**Comment by puerco** [comment]
These patches were generated using the changes submitted in  kubernetes/release#1304

File: src/assets/release-notes-1.18.1.json [hunk 1]
```diff
@@ -0,0 +1,1043 @@
+{
```
Comment (saschagrunert) [inline src/assets/release-notes-1.18.1.json:1]: I think those are too many for the 1.18.1 patch release. I guess we have to ensure that we pass the correct branch on patch releases there:
- https://github.com/kubernetes/release/blob/9a84972e209bf6f885cafe824676164cb4e04090/cmd/krel/cmd/release_notes.go#L722
- https://github.com/kubernetes/release/blob/9a84972e209bf6f885cafe824676164cb4e04090/cmd/krel/cmd/release_notes.go#L764

Comment (puerco) [reply src/assets/release-notes-1.18.1.json:1]: Ok, I'm on it. I'll merge the PR creation stuff and start fixing this one.

Comment (puerco) [reply src/assets/release-notes-1.18.1.json:1]: I've made changes to krel to better calculate te previous tag and to read from the release branch if it exists. This time we got 14 notes from 44 commits (check out the output below).

**Comment by puerco** [comment]
```
krel release-notes --format=json --tag=v1.18.1
INFO Generating release notes for tag v1.18.1     
INFO Cloning kubernetes/kubernetes                
INFO Working on branch release-1.18 instead of master 
INFO Using start tag v1.18.0                      
INFO Using end tag v1.18.1                        
INFO cloning/updating repository kubernetes/kubernetes 
INFO using found start SHA: 9e991415386e4cf155a24b1da15becaa390438d8 
INFO using found end SHA: 7879fc12a63337efff607952a323df90cdc7a335 
INFO Gathering release notes                      
INFO starting to process commit 1 of 44 (2.27%): 5dd22057dc765f5065a6c61c245534aa2b44e9b4 
INFO starting to process commit 2 of 44 (4.55%): 7879fc12a63337efff607952a323df90cdc7a335 
INFO starting to process commit 3 of 44 (6.82%): 0d2cdc3b6d506e79577adec2f1ab6a267993b593 
INFO starting to process commit 4 of 44 (9.09%): 7909fe2d9f1b3759e7c4a60bd8238cc2d9f31c55 
INFO starting to process commit 5 of 44 (11.36%): 23c6b68f842273a1cd2f1cd8733de9bbe789aba9 
INFO starting to process commit 6 of 44 (13.64%): 49aac775931dd18291d9a03dd53ce322a9a6981f 
INFO starting to process commit 7 of 44 (15.91%): 75b135452fabbc4433d4bfeb0cbb53092824cb70 
INFO starting to process commit 8 of 44 (18.18%): 6c096c579a8b85a6fa239c3a24b31a48ba6c8652 
INFO starting to process commit 9 of 44 (20.45%): a8701ff9f2f8f28bc3e300bf35f9ceea2a7437f0 
INFO starting to process commit 10 of 44 (22.73%): e9e1ea490f8e6f46730f847b2f48456098d94623 
INFO starting to process commit 11 of 44 (25.00%): 87c145b51938b69ea84fc9418104ddc96c432a7f 
INFO starting to process commit 12 of 44 (27.27%): f46ea4ddec9101ae75963677e021c652bfbef447 
INFO starting to process commit 13 of 44 (29.55%): 0dee03d1bd2764fae9a8edae69408f0cb8733908 
INFO starting to process commit 14 of 44 (31.82%): ec8c186fe8181f52670716d8d10aa7663e868491 
INFO starting to process commit 15 of 44 (34.09%): bb07a1fa1f608d6bea00264446be308ab16a276a 
INFO starting to process commit 16 of 44 (36.36%): 8fbe2f45a08c8c5769ff63bf3cb110ff70bdbbcb 
INFO starting to process commit 17 of 44 (38.64%): 98e26aff7f48eda66726e8672bb8cd1f0a7e8076 
INFO starting to process commit 18 of 44 (40.91%): fc9151c55c71f5d672729708700e43dc10e8854e 
INFO starting to process commit 19 of 44 (43.18%): b94fc6c4c47ef0dc7483d076024c48433cd96924 
INFO starting to process commit 20 of 44 (45.45%): 974a91db1c6ee85c543a6c46f03c0a0181291feb 
INFO starting to process commit 21 of 44 (47.73%): b427784213ab07731f39815bb8bc3baae4114be2 
INFO starting to process commit 22 of 44 (50.00%): 7444c1e865b64127a59f379af2c8f866d0b0f56d 
INFO starting to process commit 23 of 44 (52.27%): 4860841aa064c380714f91a1387dfc20d91b6f63 
INFO starting to process commit 24 of 44 (54.55%): 22f39eaae5f32ab78566ebf07b0ce1897253db6f 
INFO starting to process commit 25 of 44 (56.82%): 1cf2fd183b77b76b88b2e34c1eaed2837ba9456e 
INFO starting to process commit 26 of 44 (59.09%): b5470e1d6499b22efefd343828e5fcac316806f1 
INFO starting to process commit 27 of 44 (61.36%): 26a2c61bf2b28f9d4fae88fb13541bd675a58834 
INFO starting to process commit 28 of 44 (63.64%): 0c78c1e66e37b9c0c12284889a408ba5c667f321 
INFO starting to process commit 29 of 44 (65.91%): ff809a5d953ba778270ce8790b21d394821e1e28 
INFO starting to process commit 30 of 44 (68.18%): bf23e9356f01f3f17f5946c8f424d2cb6826119f 
INFO starting to process commit 31 of 44 (70.45%): d11ad211fdd1146d799eb2fbe22c88f6829c491e 
INFO starting to process commit 32 of 44 (72.73%): 262bdad7b9d2a89dcaaadb4744d76c8e7851bf8b 
INFO starting to process commit 33 of 44 (75.00%): 7e4d0319e3ad3b629bf7e171bbf7d83b82f917bf 
INFO starting to process commit 34 of 44 (77.27%): 203b1ddfdf7555c261266902d368df4333121138 
INFO starting to process commit 35 of 44 (79.55%): dd7fb533d03bab18b803203dd2d3e7fff5844aff 
INFO starting to process commit 36 of 44 (81.82%): 1b15cefcc9119c88523ca32c5918af00a26f6132 
INFO starting to process commit 37 of 44 (84.09%): 93e62e60d1d008526523ffc386c53d976f0614cb 
INFO starting to process commit 38 of 44 (86.36%): bd9b7920920c846f0cea5d2d106a129df2da65c2 
INFO starting to process commit 39 of 44 (88.64%): ac62906c24daa4029a0a94c8012247f3f77a2994 
INFO starting to process commit 40 of 44 (90.91%): d2e35c7a6a7b652331002366c68f3ca664eeedbf 
INFO starting to process commit 41 of 44 (93.18%): d5dfb5cb416fcc32ad556b0d253307c9267b1d30 
INFO starting to process commit 42 of 44 (95.45%): 38eea6b39261e14eee7f129098afd32c7437a9ce 
INFO starting to process commit 43 of 44 (97.73%): 5e251a7e39b569451a607868f6a478553a338056 
INFO starting to process commit 44 of 44 (100.00%): 9e991415386e4cf155a24b1da15becaa390438d8 
```

**Pull request merged**